katas/content/nonlocal_games/examples/CHSHGameDemo.qs

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+//////////////////////////////////////////////////////////////////////
+// This file contains reference solutions to all above exercises.
+// We recommend that you try to solve the exercises yourself first,
+// but feel free to look up the solution if you get stuck.
+//////////////////////////////////////////////////////////////////////
+
+namespace Quantum.Kata.CHSHGame {
+
```

Comment:
This is a very long code for a demo, and it will take a lot of scrolling for the learner to get to the interesting bits. Let's try to trim it down while keeping it interesting:

* Microsoft.Quantum.Intrinsic is open by default so we don't need to open it explicitly
* We don't need comments explaining the code from the exercises we've already covered or the /// headers
* We can merge CreateEntangledPair, PlayClassicCHSH, and PlayQuantumCHSH into the main operation - they made sense as separate operations in the classic kata when they were all separate tasks, but now that they're not and we need to copy all code from the previous tasks to the demo, having them as operations just takes up extra space, since they all are called just once
* Since the win probability is not 100%, the most interesting part of the demo is to show not the results of a single run, but to aggregate them across a lot of runs and show the difference of probabilities. Let's do a 1000 runs and compute the probabilities for classical and quantum outcomes for them instead of printing individual outcomes for each input

Reply:
All comments and license headers are removed from demo. Demo is refactored.